### DISCUSSION TITLE
some exes can be imported correctly with old version but not with new version?

### DISCUSSION THREAD

**Discussion by 00912** [NationalSecurityAgency/ghidra#3466] [Q&A] [answered]
not sure if it's a bug or did i do something wrong🤔
Problem: (on Windows) import some exes in 10.0.3 gives me Unexpected Loader exception but not with 9.0.4

detailed:
OS:  windows 10 version 20H2(OS Build 19042,1237)
java version "11.0.11" 2021-04-20 LTS
        Java(TM) SE Runtime Environment 18.9 (build 11.0.11+9-LTS-194)
        Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.11+9-LTS-194, mixed mode

exes i imported are from: https://github.com/Maijin/radare2-workshop-2015/tree/master/IOLI-crackme/bin-win32

import correctly with 9.0.4:
![9 0 4](https://user-images.githubusercontent.com/17740316/135384742-475fccd8-5779-4c6a-b510-de488b7fbc7f.png)
error with 10.0.3:
![10 0 3](https://user-images.githubusercontent.com/17740316/135384844-fe4f0d11-25f5-4d2d-9567-3a87d795ba7e.png)

note: crackme0x07.exe and crackme0x09.exe **do** imported correctly with 10.0.3

**Comment by ryanmkurtz** [comment, marked as answer]
This will be fixed in the upcoming 10.0.4 release.  In the meantime, uncomment this line in your `support/launch.properties` file and it should work again.
```
#VMARGS=-DContinuesInterceptor.enabled=true
```
